src/transformState.js

```diff
@@ -1,11 +1,30 @@
 'use strict';
 
-/**
- * @param {Object} state
- * @param {Object[]} actions
- */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        for (const value in action.extraData) {
+          state[value] = action.extraData[value];
+        }
```

Comment:
```suggestion
        for (const key in action.extraData) {
          state[key] = action.extraData[key];
        }
```

Reply:
if you have an object, it looks this way:
```
object[key] = value
```
so `value` is inappropriate in this context